x-pack/plugins/reporting/server/lib/esqueue/worker.js

```diff
@@ -232,7 +232,7 @@ export class Worker extends events.EventEmitter {
       const cancellationToken = new CancellationToken();
       const jobSource = job._source;
 
-      Promise.resolve(this.workerFn.call(null, job, jobSource.payload, cancellationToken))
+      Promise.resolve(this.workerFn.call(null, job, jobSource, cancellationToken))
```

Comment:
I think this should be reverted: it's from reversing code after extracting it from the PR to switch to Task Manager

Reply:
ya... not sure why we're doing a `.call` instead of just invoking it normally. We're not changing the `this` binding at all

Reply:
I seriously can not wait to get rid of this file